web/packages/teleport/src/Discover/Database/MutualTls/useMutualTls.ts

```diff
@@ -95,6 +95,7 @@ export function useMutualTls({ ctx, props }: Props) {
     canUpdateDatabase: access.edit,
     curlCmd: generateSignCertificateCurlCommand(
       clusterId,
+      '17520h',
```

Comment:
Can we move this to a const with a proper name?
At first glance, this looks like some random number :sweat_smile: